ocaml/forkexecd/lib/fe_systemctl.ml

```diff
@@ -121,15 +121,19 @@ let stop ~service =
   Xapi_stdext_unix.Unixext.unlink_safe destination ;
   status
 
-let is_active ~service =
+let check_service_status ~command ~service =
```

Comment:
The function returns a boolean; the previous `is_active` name reflected this better. Maybe just call it `status`? This would look like `status ~command:"is-enabled" ~service` at a call site. It's a minor concern, though.